pyodide-build/pyodide_build/buildall.py

```diff
@@ -200,10 +211,8 @@ def finish(self, success: bool, elapsed_time: float) -> None:
 
         self.finished = True
 
-        if success:
-            logger.success(done_message)
-        else:
-            logger.error(done_message)
+        color = SUCCESS_BLUE if success else FAIL_RED
+        print(color + done_message + END_COLOR)
```

Comment:
Using `logger.error` here just leaves an empty line on the console. I couldn't figure out how to get it to print correctly. I think this is a reasonable alternative.